### PR TITLE
feat(scss) Callout

### LINKED ISSUE #1348: [DS to LF] Messages : Callout
- [ ] Put the title in block to make the content go underneath
- [ ] Pay attention to the alignment when there is an icon
- [ ] Change the default background + palette to 100
- [ ] Add borders (200)
- [ ] Be careful with border-radius on icons

Capture d’écran 2021-05-26 à 17 32 55

### PR DESCRIPTION
**Pull request by RipripKipkip**
fixes #1348 

- [x] Put the title in block to make the content go underneath
- [x] Pay attention to the alignment when there is an icon
- [x] Change the default background + palette to 100
- [x] Add borders (200)
- [x] Be careful with border-radius on icons

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1381

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1381

File: packages/scss/src/components/_callout.scss [hunk 1]
```diff
@@ -88,8 +90,9 @@
 }
 
 @mixin calloutDefaultPalette($palette) {
-	background-color: _color($palette, 50);
+	background-color: _color($palette, 100);
 	color: _color($palette, 800);
+	border: 1px solid _color($palette, 200);
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/_callout.scss:95]: theme divider

File: packages/scss/src/theming/components/_callout.theme.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 $callout: (
   margin: 0 0 _theme("spacings.small"),
   padding: _theme("spacings.smaller") _theme("spacings.small"),
-  background: _theme("commons.background.base")
+  background: _color("grey", "100")
```
Comment (vvalentin-lucca) [inline packages/scss/src/theming/components/_callout.theme.scss:4]: background-color

File: stories/qa/callout/callout.stories.ts [hunk 3]
```diff
@@ -9,6 +9,17 @@ import { Story, Meta, moduleMetadata } from '@storybook/angular';
 export default {
   title: 'QA/Callout',
   component: CalloutStory,
+	parameters: {
+		backgrounds: {
+			default: 'toto',
```
Comment (vvalentin-lucca) [inline stories/qa/callout/callout.stories.ts:14]: ?

Comment (RipripKipkip) [reply stories/qa/callout/callout.stories.ts:14]: ah j'avais fait ça pour avoir un fond blanc je vais enlever

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1381

**Review by sanlucca — CHANGES_REQUESTED** [review]
Don't change the variable name (you can let the change on the css property)

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1381

**Review by sanlucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1381

**Pull request merged**